src/cmd/init.rs

```diff
@@ -19,6 +19,7 @@ pub enum NWords {}
 #[derive(Clone, Debug, StructOpt)]
 pub struct InitOpt {
     /// The network name (bitcoin|regtest|testnet)
+    #[structopt(name = "(bitcoin|regtest|testnet)")]
```

Comment:
```suggestion
    #[structopt(name = "bitcoin|regtest|testnet"]
```
structopt will put `<..>` around it anyways.